cuvs-bench/cpu/Dockerfile

```
@@ -31,7 +31,7 @@ mamba update --all -y -n base
 mamba install -y -n base "python=${PYTHON_VER}"
 mamba update --all -y -n base
 mamba install -y -n base \
-  "raft-ann-bench-cpu=${RAPIDS_VER}.*" \
+  "cuvs_bench_cpu=${RAPIDS_VER}.*" \
```

Update from https://github.com/rapidsai/cuvs/pull/417.
```suggestion
  "cuvs-bench-cpu=${RAPIDS_VER}.*" \
```